src/App.tsx

```diff
@@ -1,24 +1,28 @@
 import { useState } from 'react'
 import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import viteLogo from '../../../../../vite.svg'
```

Comment:
I would leave the import statement as follows: 'import viteLogo from '/vite.svg'' because the way it is currently written doesn't recognize the import (I think that you are probably importing another file which is inside your computer).
<img width="869" alt="image" src="https://github.com/sirius-valley/tricker-front/assets/112835662/5f5dfc63-0c5f-45f0-9a23-f15942333562">

Reply:
Anyway, the ESLint setup with Husky pre-commit is working correctly.